app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -1,7 +1,7 @@
 require "google/apis/drive_v3"
 
 class Publishers::Vacancies::DocumentsController < Publishers::Vacancies::BaseController
-  helper_method :form
+  helper_method :form, :step
```

Comment:
You can use `step_process.current_step` in the views I think, no need to expose this.